pkg/kubectl/cmd/cmd_test.go

```diff
@@ -221,9 +221,12 @@ func NewTestFactory() (*cmdutil.Factory, *testFactory, runtime.Codec) {
 
 func NewMixedFactory(apiClient resource.RESTClient) (*cmdutil.Factory, *testFactory, runtime.Codec) {
 	f, t, c := NewTestFactory()
+	multiRESTMapper := meta.MultiRESTMapper{}
```

Comment:
This could just be `var multiRESTMapper meta.MultiRESTMapper` to prevent an empty slice allocation. Appending to a nil slice is fine.